plugins/SLURM-Integration/ufm_slurm_utils.py

```diff
@@ -99,6 +100,10 @@ def getSlurmConfFile(self):
 
     def read_conf_file(self, key):
         conf_file = self.getSlurmConfFile()
+        conf_mode = os.stat(conf_file).st_mode
+        if conf_mode & 0o77 != 0:
+            logging.error(f"Permissions for configuration file {conf_file} are too open: {oct(conf_mode & 0o777)}")
```

Comment:
could you please tell why do you need this change?

Reply:
Please take a look at the comment in the patch:
> This file contains a password or an API token in plaintext, the administrator should be warned that this is unsafe.

Does that make sense?

It's currently too easy to setup those scripts and end up with `ufm_slurm.conf` that is readable by all cluster users, which mean they would now have access to the whole UFM REST API.